static/fetch.js

```diff
@@ -0,0 +1,49 @@
+(function(){
+	const RELOAD_TIME = 5000;
+	setInterval(loop, RELOAD_TIME);
+})();
+
+async function loop(){
+	let data = await reload();
+	for (let printer_name in data){
+		queue(printer_name, data[printer_name]);
+	}
+}
+
+function element(username, date){
+	let wrapper = document.createElement('div');
+	wrapper.classList.add('printlist-user');
+	let name = document.createElement('div');
+	name.classList.add('printlist-user-handle');
+	name.innerText = username.replace(/\</g, '&lt;').replace(/\>/g, '&gt;');
```

Comment:
I haven't really been following this project so I don't 100% know what's going on, but what are you using XML for? This line worries me, this isn't a great way to deal with XML.

Reply:
We're basically using XML to fetch requests of the most recent print jobs (so that we don't have to reload the entire page again)

Reply:
I used replace as a safety measure against XSS attacks, but looking at this again, it seems pretty unnecessary.